packages/components/src/spectrum/comboBox/ComboBox.tsx

```diff
@@ -10,7 +10,7 @@ import type { NormalizedItem } from '../utils';
 import { type PickerPropsT, usePickerProps } from '../picker';
 
 export type ComboBoxProps = PickerPropsT<SpectrumComboBoxProps<NormalizedItem>>;
-
+export { type MenuTriggerAction } from '@react-types/combobox';
```

Comment:
Aside - not our naming, but I was worried this name would conflict with something from the `@react-types/menu` package. But there they have [`MenuTriggerType`](https://github.com/adobe/react-spectrum/blob/82a4de5b01de2549e2706d048def879b8e6f12b0/packages/%40react-types/menu/src/index.d.ts#L17).
I don't know why they didn't call this `ComboBoxTriggerType` or something instead, but whatever.